e2e/playwright/testing-samples-loading.spec.ts

```diff
@@ -0,0 +1,183 @@
+import { test, expect } from '@playwright/test'
+import { getUtils, setup, setupElectron, tearDown } from './test-utils'
+import { bracket } from 'lib/exampleKcl'
+import * as fsp from 'fs/promises'
+import { join } from 'path'
+import { FILE_EXT } from 'lib/constants'
+
+test.beforeEach(async ({ context, page }, testInfo) => {
+  await setup(context, page, testInfo)
+})
+
+test.afterEach(async ({ page }, testInfo) => {
+  await tearDown(page, testInfo)
+})
+
+test.describe('Testing in-app sample loading', () => {
+  /**
+   * Note this test implicitly depends on the KCL sample "flange-with-patterns.kcl"
+   * and its title. https://github.com/KittyCAD/kcl-samples/blob/main/flange-with-patterns/flange-with-patterns.kcl
+   */
+  test('Web: should overwrite current code, cannot create new file', async ({
+    page,
+  }) => {
+    const u = await getUtils(page)
+
+    await test.step(`Test setup`, async () => {
+      await page.addInitScript((code) => {
+        window.localStorage.setItem('persistCode', code)
+      }, bracket)
+      await page.setViewportSize({ width: 1200, height: 500 })
+      await u.waitForAuthSkipAppStart()
+    })
+
+    // Locators and constants
+    const newSample = {
+      file: 'flange-with-patterns' + FILE_EXT,
+      title: 'Flange',
+    }
+    const commandBarButton = page.getByRole('button', { name: 'Commands' })
+    const commandOption = page.getByRole('option', { name: 'Open Sample' })
```

Comment:
[C]
```suggestion
    const samplesCommandOption = page.getByRole('option', { name: 'Open Sample' })
```
Small thing, just reading the code I scolled back up here to see what the `getBy..` was, could have guessed it would be something like open samples but the name was a bit vague.